TeamsProject/main.py

```diff
@@ -0,0 +1,172 @@
+import selenium
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.chrome.options import Options
+import yaml
+import sys
+import time
+import pyautogui as magic
+from selenium.common.exceptions import NoSuchElementException
+from datetime import date
+from datetime import datetime
+
+
+sys.tracebacklimit = 0
+
+
+settings_path = "D:\Desktop\Python Documents\TeamsProject\settings.yaml"
+with open(settings_path) as f:

```

Comment:
Well, not everyone has the same directory.
Pass `./settings.yaml` directly into `open()`.
Alternatively, you can use `os.path.abspath()` or pathlib's `Path().absolute()`.